src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -958,34 +957,97 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         winrt::hstring currentCommandline,
         winrt::hstring currentWorkingDirectory)
     {
+        // enumerate all the parent directories we want to import snippets from
+        std::filesystem::path directory{ std::wstring_view{ currentWorkingDirectory } };

```

Comment:
`std::wstring_view` cast can probably be safely removed.

Reply:
Surprisingly not!